ccswbs/hjckrrh
# Update LDAP module to 7.x-2.x.

LDAP 7.x-2.x is not backwards compatible with LDAP 7.x-1.x. The configuration has been updated for 2.x in the ug_user feature. One change is required in the LDAP server definition, which is usually overridden to enter the proxy account password. I have included an update hook in ug_user to update the server config to match the feature, without having to revert the feature.

Note that you *must* run database updates after pulling this update.

# Test Plan
- Log in to an existing LDAP account
- Create a Drupal account manually, and log in with the corresponding LDAP password
- Log in to an account that has no Drupal account; it should be created automatically
- In all cases, verify that edupersonaffiliation is mapped to Drupal role (staff, faculty, etc.)
